gui/next/src/routes/network/[id]/+page.svelte

```diff
@@ -43,37 +44,65 @@ $: $page.params.id && load();
 <!-- ================================================================== -->
 <style>
 
-dl {
-  margin-block-start: 1rem;
-  display: grid;
-  grid-template-columns: auto auto;
-  gap: .5em;
-  column-gap: .5em;
+:global(.log-detail-method) {
+  margin-inline-end: .2em;
+  padding: .1em .2em;
+  display: inline-block;
+  position: relative;
+  top: -1px;
+
+  border-radius: .2rem;
+  border: 1px solid var(--color-frame);
+
+  font-family: monospace;
+  font-size: .75em;
 }
 
-  dd {
-    text-align: end;
-  }
+.success {
+  color: var(--color-confirmation);
+}
+
+.error {
+  color: var(--color-danger);
+}
 
 a:hover {
   color: var(--color-interaction-hover);
 }
 
+dl {
+  margin-block-start: var(--space-page);
+}
+
 </style>
 
 
 
 <!-- ================================================================== -->
-<Aside title={$state.network.lb_status_code ? `${$state.network.http_request_method} ${$state.network.http_request_path}` : 'Loading…'} closeUrl="/network?{$page.url.searchParams.toString()}">
-
-  <dl>
-    <dt>Time:</dt> <dd>{new Date($state.network._timestamp / 1000).toLocaleString()}</dd>
-    <dt>Request path:</dt> <dd><a href="{$state.network.http_request_url}">{$state.network.http_request_path}</a></dd>
-    <dt>Request method:</dt> <dd>{$state.network.http_request_method}</dd>
-    <dt>Request protocol:</dt> <dd>{$state.network.http_request_protocol}</dd>
-    <dt>Status:</dt> <dd>{$state.network.lb_status_code}</dd>
-    <dt>Client IP:</dt> <dd>{$state.network.client}</dd>
-    <dt>Client user agent:</dt> <dd>{$state.network.user_agent}</dd>
-  </dl>
+<Aside title={$state.network.lb_status_code ? `<span class="log-detail-method">${$state.network.http_request_method}</span> ${$state.network.http_request_path}` : 'Loading…'} closeUrl="/network?{$page.url.searchParams.toString()}">
+
+  {#if $state.network._timestamp}
+    <dl class="definitions">
+
+      <dt>Time</dt> <dd>{new Date($state.network._timestamp / 1000).toLocaleString()}</dd>
+
+      <dt>Request path</dt> <dd><a href={$state.network.http_request_url}>{$state.network.http_request_path}</a></dd>
+
+      <dt>Request method</dt> <dd>{$state.network.http_request_method}</dd>
+
+      <dt>Request protocol</dt> <dd>{$state.network.http_request_protocol}</dd>
```

Comment:
Is there a different protocol than `https`?

Reply:
Can be 1 or 2. Not sure if a value information, true.

Reply:
@piotrze - removed :)